README.md

```diff
@@ -1,58 +1,117 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
-For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
-3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num". 
-For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
 
-_Bonus_ think of another way to solve it. 
-  <details>
-    <summary>
-      Hint
-    </summary>
-    Find the final number and increment the loop by 9.
-  </details>
-  
-4. Write a loop that uses console.log to log all the numbers from 1 to 100, with two exceptions. For numbers divisible by 3, log "Fizz" instead of the number, and for numbers divisible by 5 (and not 3), log "Buzz" instead.
+let num = 10
+while (num > 1){
+  console.log (num)
+  num = num - 1
+}
 
+let a = 10
+for (let num = a; num > 1; num--){
+  console.log (num)
+}
 
-5. Modify your program to log "FizzBuzz", for numbers that are divisible by both 3 and 5 (still log "Fizz" or "Buzz" for numbers divisible by only one of those).
 
+2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
+let num = 10
+let a = 0
+while ( a <= num ){
+if (a%2 === 0 ) {
+  console.log (a + " THIS IS A EVEN NUMBER")}
+  else
+  {
+    console.log (a + "This is a ODD NUMBER")
+  }
+a++
+}
+
+for (num = 10; num >= 0; num--){
+if (num%2 === 0){
+  console.log (num + " this is a even number")}
+  else
+  {
+    console.log (num + " this is a odd ")
+  }
+}
 
-Bonus: 
 
-1. Write a program that would log the lyrics of the song 99 Bottles of Beer. This is the first verse of the song:
 
-99 bottles of beer on the wall,
 
-99 bottles of beer.
+3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num".
+For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
 
```

Comment:
looking for number * 9 not 8. Also note that we wanted to see it in a string format that tells us the number we're on and says times 9 equals, and then the number you're currently logging. The Loop looks good though.